dataplane/vppagent/pkg/converter/memif_interface_converter.go

```diff
@@ -72,5 +72,15 @@ func (c *MemifInterfaceConverter) ToDataRequest(rv *rpc.DataRequest) (*rpc.DataR
 			SocketFilename: path.Join(fullyQualifiedSocketFilename),
 		},
 	})
+
+	// Process static routes
+	for _, route := range c.Connection.GetContext().GetRoutes() {
+		rv.LinuxRoutes = append(rv.LinuxRoutes, &l3.LinuxStaticRoutes_Route{
+			DstIpAddr:	route.DstIpAddr,
+			GwAddr: route.GwAddr,
```

Comment:
For the moment lets just make the route point to c.Connection.GetContext().DstIpAddr and leave off the metric

Reply:
Done.